shopelectro/models.py

```diff
@@ -232,6 +236,20 @@ def get_group_tags_pairs(self) -> List[Tuple[TagGroup, List['Tag']]]:
 
         return group_tags_pair
 
+    def create_safely(self, name: str, slug='', **kwargs):
```

Comment:
@duker33 
Maybe should we do it in `Tag.objects.save(...)` method to provide safely creation for every cases?

Reply:
hmm, i'll think about it during next day